aten/src/ATen/native/VariableMethodStubs.cpp

```diff
@@ -12,7 +12,7 @@ void backward(const Tensor& self, const Tensor& gradient, c10::optional<bool> ke
   AT_ERROR("backward is not implemented for Tensor");
 }
 
-void set_data(const Tensor& self, const Tensor& new_data) {
+void set_data(Tensor& self, const Tensor& new_data) {
```

Comment:
I'm not too clear on why these two functions (`set_data` and `retain_grad`) contained `const self` tensors to begin with, since it looks like both functions modify the tensor's state. Removing `const` here and bellow in order to keep the function declarations in line with my codegen change (see gen.py line 115)

Reply:
Yeah, I was confused too :)